.github/workflows/go.yml

```diff
@@ -10,19 +10,36 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    container: golang:1.19-bullseye
+    defaults:
+      run:
+        shell: bash # needed for codecov
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: f1refly
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: 1.19
+      - name: Fetch history # See https://github.com/actions/checkout/issues/477
```

Comment:
Out of curiosity, why did we need Git history? Is it for coverage?

Reply:
The copyright header checker - it looks for the latest year on all changed files, so if you don't have change history it thinks everything needs updating.